plugin/src/main/java/org/opensearch/ml/memory/MLMemoryManager.java

```diff
@@ -0,0 +1,13 @@
+package org.opensearch.ml.memory;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+/**
+ * Manager class for Memory related calls. It contains ML memory related operations like CRUD Conversation/Interaction etc.
+ */
+@Log4j2
+@AllArgsConstructor
+public class MLMemoryManager {
+
+}
```

Comment:
What does this do?

Reply:
This shouldn't be included in this PR. I will remove it. It's for the plugin to query memory with different algorithms.